recipes/trf/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: trf
+  version: "4.07b"
+
+source:
+  fn: trf407b
+  url: https://tandem.bu.edu/trf/downloads/trf407b.linux64 # [linux]
+  url: https://tandem.bu.edu/trf/downloads/trf407b.macos64 # [osx]
```

Comment:
Than it's ok ... maybe you can ask them for the source and enhance this recipe in build-1 if you get the source.